modules/store/migrations/migration.json

```diff
@@ -20,6 +20,11 @@
       "description": "The road to v13 beta",
       "version": "13-beta",
       "factory": "./13_0_0-beta/index"
+    },
+    "ngrx-store-migration-13-0-1": {
+      "description": "The road to v13.0.1",
```

Comment:
@david-shortman FYI, version `13.0.0` is not published yet. This migration should be included in `v13.0.0-rc.1` or `v13.0.0`.

cc @brandonroberts